BackEnd/src/main/java/com/javaschool/service/QueueResponseServiceImp.java

```diff
@@ -0,0 +1,39 @@
+package com.javaschool.service;
+
+import com.javaschool.entitymapper.PackageType;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.List;
+
+@Service
+public class QueueResponseServiceImp implements QueueResponseService{
+
+    private static final Logger logger = LoggerFactory.getLogger(QueueSenderServiceImp.class);
```

Comment:
The naming convention for constants are UPPER_CASE
Please change `logger` to `LOGGER`